### PR TITLE
S-21 to master

### PR DESCRIPTION
**Pull request by obabiy**

### CODE REVIEW

**Review by vansisto — APPROVED** [review]

File: src/main/java/com/intentsg/shop/controllers/CartControllerRest.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+import com.intentsg.shop.model.Cart;
+import com.intentsg.shop.services.CartService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/cartRest")
```
Comment (PetroRavlinko) [inline src/main/java/com/intentsg/shop/controllers/CartControllerRest.java:11]: Not plural

**Review by PetroRavlinko — CHANGES_REQUESTED** [review]
It's it rest api. Map requests in right REST way

File: src/main/java/com/intentsg/shop/controllers/CartControllerRest.java (hunk 1 above)
Comment (IvanOlkhoviy) [reply src/main/java/com/intentsg/shop/controllers/CartControllerRest.java]: Fixed

File: src/main/java/com/intentsg/shop/controllers/CartControllerRest.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+import com.intentsg.shop.model.Cart;
+import com.intentsg.shop.services.CartService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/cartsRest")
```
Comment (PetroRavlinko) [inline src/main/java/com/intentsg/shop/controllers/CartControllerRest.java:11]: just carts

**Pull request closed without merging**